packages/http/src/index.ts

```diff
@@ -7,7 +7,14 @@ export * from "./decorators.js";
 export * from "./metadata.js";
 export * from "./operations.js";
 export * from "./parameters.js";
-export { getHttpFileModel, isHttpFile, isOrExtendsHttpFile } from "./private.decorators.js";
+export {
+  HttpPart,
+  HttpPartOptions,
+  getHttpFileModel,
+  getHttpPart,
+  isHttpFile,
+  isOrExtendsHttpFile,
+} from "./private.decorators.js";
```

Comment:
I notice this is in private decorator, should we move them out of private if we have public usage?